packages/core/src/lib/collapse.svelte

```diff
@@ -0,0 +1,95 @@
+<!--
+@component
+  
+An element that Toggles visibility of content.
+
+```svelte
+    <Collapse
+      title='Motor 1'
+      variant="minimal"
+    >
+      <Breadcrumbs slot='title' crumbs={['Robot', 'Motor']}></Breadcrumbs>
+      <Badge slot='header' label='Inactive'></Badge>
+      <div style='font-size: 12px; padding: 1rem'>Motor one was concieved and executed at Bell Labs in 1972 under the guidance of lead director Dennis Richie and Superviser Wallace Breen.</div>
+    </Collapse>
+```
+-->
+<svelte:options immutable />
+
+<script lang="ts">
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Icon } from '$lib';
+
+type Variants = 'default' | 'minimal';
+
+/**
+ * The title for the collapse component.
+ */
+export let title = '';
+/**
+ * Whether the collapse is in the open position.
+ */
+export let open = false;
+/**
+ * The variant of the collapse component.
+ */
+export let variant: Variants = 'default';
+
+const dispatch = createEventDispatcher();
```

Comment:
```suggestion
type Events = {
  /** Fires when the collapse is toggled */
  toggle: boolean
}

const dispatch = createEventDispatcher<Events>();

Reply:
We should update all components to this event dispatcher style.